Makefile

```diff
@@ -1316,7 +1318,7 @@ test_scenario_%:
 	$(Q) $(COMPOSER_TESTS) scenario $*
 
 merge_coverage_reports:
-	$(PHPCOV) merge --clover reports/coverage.xml reports/cov
+	php -d memory_limit=-1 $(PHPCOV) merge --clover reports/coverage.xml reports/cov
```

Comment:
The reason behind `memory_limit=-1` here is the number of `.cov` files being processed. I'm talking about 100s of coverage files being merged (1 per request + 1 per test suite).